web_widget_mermaid/__init__.py

```diff
@@ -0,0 +1 @@
+# -*- coding: utf-8 -*-
```

Comment:
not needed
Add `License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)`